openmetadata-ui/src/main/resources/ui/src/pages/AddObservabilityPage/DestinationFormItem/DestinationFormItem.component.tsx

```diff
@@ -37,61 +30,51 @@ function DestinationFormItem({
     [];
 
   return (
-    <Card className="alert-form-item-container">
-      <Row gutter={[8, 8]}>
-        <Col span={24}>
-          <Typography.Text className="font-medium">{heading}</Typography.Text>
-        </Col>
-        <Col span={24}>
-          <Typography.Text className="text-xs text-grey-muted">
-            {subHeading}
-          </Typography.Text>
-        </Col>
-        <Col span={24}>
-          <Form.List
-            name={['destinations']}
-            rules={[
-              {
-                validator: listLengthValidator(t('label.destination')),
-              },
-            ]}>
-            {(fields, { add, remove }, { errors }) => {
-              return (
-                <Row
-                  data-testid="destination-list"
-                  gutter={[16, 16]}
-                  key="destinations">
-                  {fields.map(({ key, name }) => (
-                    <DestinationSelectItem
-                      id={name}
-                      key={key}
-                      remove={remove}
-                      selectorKey={key}
-                    />
-                  ))}
+    <FormCardSection
+      heading={t('label.destination')}
+      subHeading={t('message.alerts-destination-description')}>
+      <Form.List
+        name={['destinations']}
+        rules={[
+          {
+            validator: listLengthValidator(t('label.destination')),
+          },
+        ]}>
+        {(fields, { add, remove }, { errors }) => {
+          return (
+            <Row
+              data-testid="destination-list"
+              gutter={[16, 16]}
+              key="destinations">
+              {fields.map(({ key, name }) => (
+                <DestinationSelectItem
+                  id={name}
+                  key={key}
+                  remove={remove}
+                  selectorKey={key}
+                />
+              ))}
 
-                  <Col span={24}>
-                    <Button
-                      data-testid="add-destination-button"
-                      disabled={
-                        isEmpty(selectedSource) || isNil(selectedSource)
-                      }
-                      type="primary"
-                      onClick={() => add({})}>
-                      {buttonLabel}
-                    </Button>
-                  </Col>
+              <Col span={24}>
+                <Button
+                  data-testid="add-destination-button"
+                  disabled={isEmpty(selectedSource) || isNil(selectedSource)}
+                  type="primary"
+                  onClick={() => add({})}>
```

Comment:
```suggestion
                  onClick={add}>
```
Not sure if passing empty object is necessary here?